includes/admin.form.inc

```diff
@@ -39,6 +39,13 @@ function islandora_basic_collection_admin(array $form, array &$form_state) {
           '#default_value' => variable_get('islandora_basic_collection_page_size', '12'),
           '#description' => t('The default number of objects to show in a collection view.'),
         ),
+        'islandora_basic_collection_admin_page_size' => array(
+          '#type' => 'textfield',
+          '#title' => t('Default collection objects per page in the admin view'),
```

Comment:
@rosiel

I think `during collection management` is preferable to `in the admin view` as it is more descriptive without adding much length as you will be able to eliminate the other usage of `collection`. What do you think about that change?

Thanks!

----
William Panting
_Developer_
*[discoverygarden inc.](http://www.discoverygarden.ca) | Managing Digital Content*